### ISSUE TITLE
SpatialEntity throwing error

### ISSUE THREAD

**Issue by FlashHit** [Imposter/vscode-lua-vu#8]
`ext\Shared\.vu\types\shared\type\SpatialEntity.lua`

**Comment by FlashHit** [comment]
```
---@class SpatialEntity:Entity
---@field transform LinearTransform @ Gets or sets the world transform of this entity.
  > NOTE: You must assign a new LinearTransfrom to update, changing individual vector components **will not work**

Example:
` ` `lua
local myTransform = mySpatialEntity.transform
myTransform.trans.y = 1
mySpatialEntity.transform = myTransform
` ` `

---@field aabb AxisAlignedBox @ **Read Only**
---@field aabbTransform LinearTransform @ **Read Only**
---SpatialEntity (Class)
SpatialEntity = {}
```

**Comment by Imposter** [comment]
Ahh, I thought I had fixed this issue with multi-line comments. I will fix this in the next update.

**Comment by Imposter** [comment]
Fixed in commit f7782a7
